assets/src/scss/layout/navbar/_search.scss

```diff
@@ -102,6 +102,10 @@
     color: inherit;
     width: calc(100% - 48px);
   }
+
+  &::placeholder {
```

Comment:
We kept the browser default here before, but Houssam requested this color, since it's also part of our current colors (not only the new identity ones) I assume we can make the change right away